tests/wrappers/test_record_episode_statistics.py

```diff
@@ -74,29 +73,3 @@ def test_wrong_wrapping_order():
 
     with pytest.raises(AssertionError):
         wrapped_env.step(wrapped_env.action_space.sample())
-
-
-def test_add_vector_episode_statistics():
```

Comment:
Why have you removed this test? 
If it fails, could you fix it or replace it with an equivalent test

Reply:
If I remove the use of the function in RecordEpisodeStatistics, it is used nowhere else. If I delete the function, I suppose I could add a test that checks the returned EpisodeStatistics info object?

Reply:
Your right, I don't think we need to replace this function with anything